supabase/migrations/06_connectors.sql

```diff
@@ -3,15 +3,16 @@
 create table connectors (
   like internal._model including all,
 
-  external_url         text not null,
-  image_name           text unique not null,
-  open_graph           jsonb_obj
+  external_url           text not null,
+  image_name             text unique not null,
+  open_graph             jsonb_obj
     generated always as (internal.jsonb_merge_patch(open_graph_raw, open_graph_patch)) stored,
-  open_graph_raw       jsonb_obj,
-  open_graph_patch     jsonb_obj,
-  oauth2_client_id     text,
-  oauth2_client_secret text,
-  oauth2_spec          jsonb_obj,
+  open_graph_raw         jsonb_obj,
+  open_graph_patch       jsonb_obj,
+  oauth2_client_id       text,
+  oauth2_client_secret   text,
+  oauth2_injected_values jsonb_obj,
```

Comment:
It's fine to do this migration manually this time. Just a heads up that in the future I'd like us to stick with the additive migration approach, and never modify a migration that's already been run.

Reply:
This SQL file is intended to be pristine and it's correct to update it in place here. Instead, there ought to be a separate migration in `supabase/pending/` to be applied to the prod DB.